src/main/java/de/ait/secondlife/domain/dto/OfferToDraftDto.java

```diff
@@ -0,0 +1,19 @@
+package de.ait.secondlife.domain.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+import lombok.ToString;
+
+@Data
+@ToString
+public class OfferToDraftDto {
```

Comment:
Maybe need to rename class to distinguish common Draft and Draft with rejected reasons. Maybe we can name second one as "RejectedOfferToDraftDto"